hack/make-release-artifacts.sh

```diff
@@ -46,7 +46,7 @@ done < <(find "${bin_dir}" -print0)
 echo >&2 "Creating ${krew_tar_archive} archive."
 (
   cd "${bin_dir}"
-  GZIP=-n tar czvf "${SCRIPTDIR}/../out/${krew_tar_archive}" ./*
+  tar -I "gzip -${n}" -cvf "${SCRIPTDIR}/../out/${krew_tar_archive}" ./*
```

Comment:
I think the `-n` is just an option and should not be read as an env var:
```suggestion
  tar -I "gzip -n" -cvf "${SCRIPTDIR}/../out/${krew_tar_archive}" ./*
```

Can you also add a `--best` gzip option?

Reply:
Right, updated.